test/unit/models/embassy_test.rb

```diff
@@ -56,4 +56,32 @@ class EmbassyTest < ActiveSupport::TestCase
     assert_equal "Legoland", location.remote_services_country.name
     assert_equal "British Embassy Legoland", location.offices.first.title
   end
+
+  test '#can_assist_british_nationals? returns true if the world location is a special case' do
+    location_name = 'Central African Republic'
```

Comment:
Minor: Perhaps using `Embassy::SPECIAL_CASES.keys.first` would make things clearer here and avoid the need to make an assertion against the test setup, i.e. `assert Embassy::SPECIAL_CASES.has_key?(location_name)` (below).

Reply:
Addressed in 93b235c1d0